src/components/button/button.tsx

```diff
@@ -81,7 +81,7 @@ export class Button {
                     'just-failed': this.justLoaded && this.loadingFailed,
                     'mdc-button--outlined': this.outlined,
                 }}
-                disabled={this.disabled}
+                disabled={this.disabled || this.loading}
```

Comment:
I added this, following @vicgeralds' suggestion [here](https://github.com/Lundalogik/lime-elements/pull/1687#discussion_r839490829).